alena_automation/XPathExercises - Alena2.csv

```diff
@@ -0,0 +1,40 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces),
+Normal:,,,
+all elements,/authors,,
+"all ""author"" elements",//author,,
+"the root element ""authors"" - 2 variants","/* ,  authors, child::authors",,
+"the same as 3,  but this time without using the name of the element",/*,,
+all elements enclosed by elements  /authors/author,//author/*,,
+"all ""name"" elements (without using names of parent elements) ",//name,,
+all elements which have 1 ancestors,//preceding::*   ,,
+"all ""period"" attributes of ""author"" elements - 2 variants","//@period , ",,
```

Comment:
**author** should be present in xpath, if there were "period" attributes in other nodes (not only in author) your xpath will return all of them, but task requires only period elements from author

Reply:
also please think of 2nd variant